report/marshal_test.go

```diff
@@ -71,3 +73,51 @@ func TestBiggerRoundtrip(t *testing.T) {
 		t.Errorf("%v != %v", r1, *r2)
 	}
 }
+
+func TestControlsCompat(t *testing.T) {
+	testData := `{
+  "Container": {
+    "nodes": {
+      "031d;<container>": {
+        "id": "031d;<container>",
+        "latest": {
+          "control_probe_id": {
+            "timestamp": "2019-10-14T14:36:01Z",
+            "value": "29b4f381044a89a3"
+          }
+        },
+        "latestControls": {
+          "docker_attach_container": {
+            "timestamp": "2019-10-14T14:36:01Z",
+            "value": {"dead": true}
+          },
+          "docker_remove_container": {
+            "timestamp": "2019-10-14T14:36:01Z",
+            "value": {"dead": false}
+          }
+        },
```

Comment:
Could you add a couple of more non-dead entries here? e.g. some with the same control name but different timestamps and some other with different control names for a more complete test?